components/playlist/playlist_types.h

```diff
@@ -36,7 +38,19 @@ struct PlaylistChangeParams {
 };
 
 struct PlaylistItemInfo {
+  struct TitleTag {};
+  using Title = base::StrongAlias<struct TitleTag, std::string>;
+
+  struct ThumbnailPathTag {};
+  using ThumbnailPath = base::StrongAlias<struct ThumbnailPathTag, std::string>;
+
+  struct MediaFilePathTag {};
+  using MediaFilePath = base::StrongAlias<struct MediaFilePathTag, std::string>;
```

Comment:
why declare `struct MediaFilePathTag {};`? looks like we can just write something like:
`using MediaFilePath = base::StrongAlias<class MediaFilePathTag, std::string>;`
it acts as a forward decl automatically, so you don't need to actually declare the class.

Reply:
I think I've seen linking errors without tag structs, but I seem to be wrong. Thanks!